logr.go

```diff
@@ -365,10 +411,33 @@ type LogSink interface {
 // This is an optional interface and implementations are not required to
 // support it.
 type CallDepthLogSink interface {
-	// WithCallDepth returns a Logger that will offset the call stack by the
-	// specified number of frames when logging call site information.  If depth
-	// is 0 the attribution should be to the direct caller of this method.  If
-	// depth is 1 the attribution should skip 1 call frame, and so on.
+	// WithCallDepth returns a log sink that will offset the call stack by the
+	// specified number of frames when logging call site information.
+	//
+	// If depth is 0, the log sink should skip exactly the number
+	// of call frames defined in RuntimeInfo.CallDepth when Info
+	// or Error are called, i.e.  the attribution should be to the
+	// direct caller of Logger.Info or Logger.Error.
+	//
+	// If depth is 1 the attribution should skip 1 call frame, and so on.
 	// Successive calls to this are additive.
 	WithCallDepth(depth int) LogSink
 }
+
+// HelperLogSink represents a Logger that knows how to climb the call stack
+// to identify the original call site and can skip intermediate helper functions
```

Comment:
Hard to tell but these lines look like thy might be > 80 columns?  I have tried to keep comments wrapped cleanly